programs/sll_sum_lists.py

```diff
@@ -0,0 +1,57 @@
+# Sum lists (page 95. 2.5)
+# input 2 sll exem. (2 -> 1 -> 6)  (2 -> 5 -> 3)   its equal to 612+352 = 964
+# return the sum as a sll ( 4 -> 6 -> 9)
+
+from single_linked_lists import Node
+
+
+def sum_nodes(a: Node, b: Node):
+    print(a)
+    print(b)
```

Comment:
Don't do prints like this in the answer unless they are part of the answer. Move them to before you call the function in line 55